.github/workflows/flatpak.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+name: Flatpak
```

Comment:
Name this "Portal Test Flatpak", more clear what it is.

Reply:
Sure